verifiers/utils/dsse.go

```diff
@@ -42,13 +43,17 @@ func PayloadFromEnvelope(env *dsselib.Envelope) ([]byte, error) {
 	return payload, nil
 }
 
+// StatementFromBytes parses the provided byte slice as a JSON payload and returns an intoto.Statement.
+// Ideally, we use the "V1" Statement in https://pkg.go.dev/github.com/in-toto/attestation/go/v1#pkg-constants,
+// but it parses json fields in snake case, while the official spec uses camel case
+// https://github.com/in-toto/attestation/blob/v1.0/spec/v1.0/statement.md.
 func StatementFromBytes(payload []byte) (*intoto.Statement, error) {
 	var statement intoto.Statement
 	if err := json.Unmarshal(payload, &statement); err != nil {
 		return nil, fmt.Errorf("%w: %w", serrors.ErrorInvalidDssePayload, err)
 	}
 
-	if statement.Type != intoto.StatementInTotoV01 {
+	if statement.Type != intoto.StatementInTotoV01 && statement.Type != intotoAttestattions.StatementTypeUri {
```

Comment:
Can we use `intot.StatementInTotoV1` to avoid pulling in another package? https://github.com/in-toto/in-toto-golang/blob/72262c0fd3a9c392440040c04336174eddac427b/in_toto/attestations.go#L18C2-L18C19

Reply:
That's not yet released in the latest v0.9.0, but that would be ideal
 - https://github.com/in-toto/in-toto-golang/blob/v0.9.0/in_toto/attestations.go#L10-L22

Reply:
Ah that makes sense. We could also hardcode the string, but this LGTM as well.